src/frontend/components/DashboardVideoLiveControlPane/inputs/DashboardVideoLiveWidgetTextAreaInput/index.spec.tsx

```diff
@@ -0,0 +1,53 @@
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import React from 'react';
+
+import { DashboardVideoLiveWidgetTextAreaInput } from '.';
+
+let inputTextValue: string;
+let nbrOfCall = 0;
+const setValueMock = (value: string) => {
+  inputTextValue += value;
+  nbrOfCall++;
+};
+
+describe('<DashboardVideoLiveWidgetTextAreaInput />', () => {
+  beforeEach(() => {
+    jest.resetAllMocks();
+    nbrOfCall = 0;
+    inputTextValue = '';
+  });
+
+  it('renders the text area with empty string and placeholder and then types some text', () => {
+    render(
+      <DashboardVideoLiveWidgetTextAreaInput
+        placeholder="An example placeholder"
+        setValue={setValueMock}
+        title="An example title"
+        value={''}
+      />,
+    );
+
+    screen.getByPlaceholderText('An example placeholder');
+
+    const textInput = screen.getByRole('textbox', { name: 'An example title' });
+    userEvent.type(textInput, 'An example typed text');
+
+    expect(nbrOfCall).toEqual('An example typed text'.length);
+    expect(inputTextValue).toEqual('An example typed text');
+  });
+
+  it('renders the text area with text and placeholder', () => {
+    render(
+      <DashboardVideoLiveWidgetTextAreaInput
+        placeholder="An example placeholder"
+        setValue={setValueMock}
+        title="An example title"
+        value={'An example typed text'}
+      />,
+    );
+
+    const textInput = screen.getByRole('textbox', { name: 'An example title' });
+    expect(textInput).toHaveValue('An example typed text');
```

Comment:
you should add the following test to match the title of your test
```
it('renders the text area with text and **placeholder**', () => {
```
```
screen.getByPlaceholderText('An example placeholder');

Reply:
I changed the test's title for `it renders the text area with text` because we already discussed about it and another PR and stated it was less confusing this way, but I also added the placeholder test